### ISSUE TITLE
[HW4] csrf token 검증 방식

### ISSUE THREAD

**Issue by syrius527** [swsnu/swppfall2022#168]
안녕하세요 조교님 csrf token을 검증하는 방식에 대한 질문이 있습니다.
우선은 postman으로 테스트하는 과정에서 스펙에 명시된 대로 'GET  /api/token/'으로 token을 가져온 다음 POST와 PUT request들의 header에 HTTP_X_CSRFTOKEN으로 넣어주려고 했으나 계속 403 오류로 인해 발급받은 csrf token으로 검증이 불가능했습니다.
아래 링크대로 postman에서 header에 HTTP_X_CSRFTOKEN 대신 X-CSRFTOKEN으로 넣어서 시도해도 되나요?
https://github.com/swsnu/swppfall2020/issues/164
또 POST, PUT request에 대해 코드 상 header에 csrf token을 추가하는 로직을 구현할 필요 없이 postman에서 header에 넣어줬을 때 잘 돌아가면 되는건가요?

**Comment by SungwonShim** [comment]
1번 질문의 경우 제가 postman으로 실험했을때는 말씀하신대로 X- 로만 했을때도 실행되었습니다!

**Comment by syrius527** [comment]
> 1번 질문의 경우 제가 postman으로 실험했을때는 말씀하신대로 X- 로만 했을때도 실행되었습니다!

네 저도 HTTP_X_는 동작 안하는데 X-로만 동작하는데 스펙에 나와있지 않아 여쭤봤습니다ㅎㅎ
감사합니다!!

**Comment by jaewooMaeng** [comment]
네 X-CSRFTOKEN로 하셔도 괜찮습니다. csrf logic 또한 생각하신대로 postman에서 넣어줬을 때 잘 돌아가면 괜찮습니다.
